### DISCUSSION TITLE
Vodafone Cable Germany specific: lowest ping time ever

### DISCUSSION THREAD

**Discussion by A5005** [jamulussoftware/jamulus#1849] [Technical Support & Help]
Hello, I've been looking for a long time for an optimal server that works best with my Vodafone cable connection in Berlin Germany. Unfortunately, Berlin is very far from the Frankfurt hub.

Please also https://sourceforge.net/p/llcon/discussion/533517/thread/b6c1900d8e/)

On 'Genres 1' I noticed the server  'MonkeyTrip Public Se' which is always at the top with 10ms ping and has only a few dropouts when jamming. I never saw ping times around 10ms before.

If you trace the IP of the server with https://hostingchecker.com/ you get the following results:

It is hosted by: Comunitel Global
Organization name: Comunitel Global S.A.
IP address: 95.62.57.16
AS (autonomous system) number and organization: AS12430 VODAFONE ESPANA S.A.U.
AS name: VODAFONE_ES
Reverse DNS of the IP:
City: A Coruña
Country: Spain

A Spanish provider ? ...but why does it work so well for me in Germany? It must be due to the provider Vodafone. I am not very familiar with hosters att all, but according to my research you cannot rent servers from Vodafone as a private customer with an acceptable price. http://www.vodafone.com.eg/VIS_templates/static/BusinessSectionAllPages/CloudVirtualServer-en.html

Does anyone have an idea to rent a server like MonkeyTrip Public Se or have good ping times with  Vodafone cable ?

**Comment by gene96817** [comment]
Some ISPs now offer hosting services.
More interesting, try a traceroute to the server. How many routers are there between you and the server?

**Reply by A5005** [reply]
Thanks for the answer. As I said, I don't know that much about all this stuff. Can the trace give information about a suitable server hoster?
![tracer](https://user-images.githubusercontent.com/44929101/121187009-e723c580-c867-11eb-9554-cbe1111cbf61.png)

**Comment by dakhubgit** [comment]
Light speed is 300km/ms one-way.  10ms Berlin/Spain for a roundtrip sounds rather unrealistic using most high-bandwidth technology.

**Comment by passing** [comment]
I think you are mixing up things, as there's 2 "MonkeyTrip" Servers on the Genre 1 List (see https://explorer.jamulus.io/):
```
🐵🎸MonkeyTrip🎸🐵	Madrid	Spain	0 / 10	46	95.62.57.16
MonkeyTrip Public Se	Germany	Germany	0 / 10	23	81.169.195.173
```
The 2nd server is hosted at Strato and so probably in their datacenter in Berlin

**Reply by A5005** [reply]
sorry, it was the wrong ip. I think I will try out a strato server, it is a well known German provider.


![tracer2](https://user-images.githubusercontent.com/44929101/121188518-7b425c80-c869-11eb-8037-d43b38aaaf0b.png)
